code/missionui/missiondebrief.cpp

```diff
@@ -1835,6 +1835,9 @@ void debrief_text_init()
 		}
 	}
 
+	// MageKing17 - Set the font so that wordwrapping in brief_color_text_init() calculates based on the same font as the debriefing itself.
```

Comment:
I'm pretty sure I wasn't the first person to add a font-setting line to the codebase... but regardless, leaving this comment untouched is just being cruel to future `git blame`-ers. :P